lib/vsc/administration/slurm/sync.py

```diff
@@ -338,7 +349,13 @@ def slurm_user_accounts(vo_members, active_accounts, slurm_user_info, clusters,
             ])
 
             # these are the current Slurm users per Account, i.e., the VO currently being processed
-            slurm_acct_users = set([user for (user, acct) in cluster_users_acct if acct == vo_id])
+            slurm_acct_users = set([user for (user, def_acct, _) in cluster_users_acct if def_acct == vo_id])
+
+            # these are the users for which an earlier delete failed because they still had live job in their (at that
```

Comment:
says who? these users are part of this set, but there might be others to, no?

Reply:
No, we grab them all. AFAICT.

Reply:
Maybe I misunderstood. What others and where?

Reply:
- By default, we only have one account per user. 
- When a user switches VO, he will be added to the new VO and the default account will be set to the new VO. He might not be removed from the old. 
- When we look through the associations, I believe we find all those that need to be removed. If they fail to be removed due to running jobs, they will be picked up again the next run. New jobs should be submitted under the default account, thus the jobs under the new association will eventually be gone.

What am I missing?